tests/integration/test_async_insert_adaptive_busy_timeout/test.py

```diff
@@ -344,29 +344,34 @@ def test_change_queries_frequency():
 
     settings = copy.copy(_query_settings)
     min_ms = 50
+    max_ms = 200
+
     settings["async_insert_busy_timeout_min_ms"] = min_ms
-    settings["async_insert_busy_timeout_max_ms"] = 2000
+    settings["async_insert_busy_timeout_max_ms"] = max_ms
 
-    _insert_queries_in_parallel(
+    _insert_queries_sequentially(
         table_name,
         settings,
-        thread_num=15,
-        tasks=2000,
+        iterations=50,
         max_values_size=1000,
-        array_size_range=[10, 15],
+        array_size_range=[10, 50],
     )
 
-    _insert_queries_sequentially(
+    select_log_query = f"SELECT countIf(timeout_milliseconds - {min_ms} < 25) FROM (SELECT timeout_milliseconds FROM system.asynchronous_insert_log ORDER BY event_time DESC LIMIT 10)"
+    res = node.query(select_log_query)
```

Comment:
Please `node.query("SYSTEM FLUSH LOGS")` before selecting from the log. If the test run order is not determined, the assert can fail because there are no recent entries in the table.
(I should've done it in the first place).